### PR TITLE
go1.16 deps

### PR DESCRIPTION
**Pull request by blixenkrone**
**WHY**
We need to have go v.16 in the go-kit pipeline for github actions

### CODE REVIEW

File: go1.16/test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM golang:1.15-alpine
```
Comment (lukeprentice202) [inline go1.16/test/Dockerfile:1]:
```suggestion
FROM golang:1.16-alpine
```

File: README.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ jesseobrien/golang-pipeline/go1.13/release@master
 jesseobrien/golang-pipeline/go1.14/release@master
 # Run release in Go1.15
 jesseobrien/golang-pipeline/go1.15/release@master
+# Run release in Go1.16
+jesseobrien/golang-pipeline/go1.15/release@master
```
Comment (vishen) [inline README.md:39]: go1.15 -> go1.16 in the repo link?

File: go1.16/test/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine
+
+LABEL name="Golang Pipeline"
+LABEL maintainer="jesseobrien"
+LABEL version="0.2.9"
+LABEL repository="https://github.com/jesseobrien/golang-pipeline"
```
Comment (vishen) [inline go1.16/test/Dockerfile:6]: Should we also change these from jessobrien to flypay?

**Review by vishen — APPROVED** [review]

File: go1.16/test/Dockerfile (hunk 3 above)
Comment (blixenkrone) [reply go1.16/test/Dockerfile]: 215d50c

File: README.md (hunk 2 above)
Comment (blixenkrone) [reply README.md]: 215d50c

**Pull request merged**